src/libraries/Microsoft.Extensions.Logging.Console/src/Resources/Strings.resx

```diff
@@ -120,4 +120,10 @@
   <data name="BufferMaximumSizeExceeded" xml:space="preserve">
     <value>Cannot allocate a buffer of size {0}.</value>
   </data>
+  <data name="BufferModeNotSupported" xml:space="preserve">
+    <value>{0} is not a supported buffer mode value.</value>
+  </data>
+  <data name="WarningMessageOnDrop" xml:space="preserve">
+    <value> message(s) dropped because of queue size limit. Increase the queue size or decrease logging verbosity to avoid this. You may change `ConsoleLoggerBufferFullMode` to stop dropping messages.</value>
```

Comment:
```suggestion
    <value> message(s) dropped because of queue size limit. Increase the queue size or decrease logging verbosity to avoid this. You may change `ConsoleLoggerQueueFullMode` to stop dropping messages.</value>
```